Packs/CortexAttackSurfaceManagement/Scripts/RankServiceOwners/RankServiceOwners.yml

```diff
@@ -5,6 +5,16 @@ args:
 - description: System IDs or names associated with the compute instance.
   isArray: true
   name: asmsystemids
+- defaultValue: asmserviceowner
+  description: The field of the alert or issue that owners should be stored.
```

Comment:
Perhaps this should be: The field of the alert or issue in which the owners should be stored.

Reply:
fixed in [changed wording](https://github.com/demisto/content/pull/38091/commits/da695d102014a16de9148a82830dec467397e407)